src/modules/Security.rb

```diff
@@ -938,6 +943,24 @@ def import_lsm_config(settings)
       PackagesProposal.SetResolvables("LSM", :pattern, lsm_config.needed_patterns)
     end
 
+    # It enables the security policies according to the profile
+    #
+    # @param settings [Hash] security settings to import from the profile
+    def import_security_policies(settings)
+      return unless settings["security_policies"].is_a?(Array)
```

Comment:
So invalid profiles will be quietly skipped here? And reported elsewhere?

Reply:
We extended the AutoYaST schema (see the [rnc](https://github.com/yast/yast-security/pull/128/files#diff-24ff10d70e0aef4b1cc92578c7a95750ab4f44e099f7a850fe06c3e322596a73) and https://github.com/yast/yast-schema/pull/145), so you get a validation error at the beginning of the auto-installation.